h5c/vic/src/vic-webapp/src/app/shared/utils/vch/vch-utils.ts

```diff
@@ -155,25 +156,18 @@ export function processPayloadFromUiToApi(payload): VchApi {
 
       if (payload.networks.clientNetworkIp) {
         vchNetwork.client.static = payload.networks.clientNetworkIp;
-        /*{
-          ip: payload.networks.clientNetworkIp
-        };*/
 
         vchNetwork.client.gateway = {
           address: payload.networks.clientNetworkGateway
         };
 
         if (payload.networks.clientNetworkRouting && payload.networks.clientNetworkRouting.length) {
-          vchNetwork.client.gateway.routing_destinations = payload.networks.clientNetworkRouting;
-        }
-
-        if (payload.networks.dnsServer && payload.networks.dnsServer.length) {
-          vchNetwork.client.nameservers = payload.networks.dnsServer;
+          vchNetwork.client.gateway.routing_destinations = payload.networks.clientNetworkRouting.split(',');
         }
       }
     }
 
-    if (payload.networks.managementNetwork) {
+    if (payload.networks.managementNetwork && payload.networks.managementNetwork !== '') {
```

Comment:
wouldn't checking if(payload.networks.managementNetwork) suffice for checking empty string?